crates/teloxide-core/src/types/reply_keyboard_markup.rs

```diff
@@ -125,7 +125,7 @@ impl KeyboardMarkup {
     /// Sets [`selective`] to `true`.
     ///
     /// [`selective`]: KeyboardMarkup::selective
-    pub fn selective<T>(self) -> Self {
+    pub fn selective(self) -> Self {
```

Comment:
If this function is public, we should probably document the change in `CHANGELOG.md`.